packages/Button/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Changelog
+
+#### Breaking Changes
+#### Improvements
+- [98] (https://github.com/mesg-foundation/mesg-components/pull/98/) Refactoring of the scss.
```

Comment:
Just a tiny formatting issue

This should be:
```
- [#98](https://github.com/mesg-foundation/mesg-components/pull/98/) Refactoring of the scss.
```

`#` in front of the number of the pull request and space to make it a link.
Same for all the other changelogs

Reply:
Fixed